AWSSDKSwiftCLI/Sources/AWSCLIUtils/Version.swift

```diff
@@ -55,9 +82,7 @@ public extension Version {
        Version(
             self.major + 1,
             0,
-            0,
-            prereleaseIdentifiers: prereleaseIdentifiers,
-            buildMetadataIdentifiers: buildMetadataIdentifiers
+            0
```

Comment:
Prerelease & build metadata identifiers are removed, since our project does not use them.